webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -263,30 +263,33 @@ protected function execute(InputInterface $input, OutputInterface $output)
         pcntl_signal(SIGTERM, [$this, 'stopCommand']);
         pcntl_signal(SIGINT, [$this, 'stopCommand']);
 
+        if (!$this->readConfig($input->getArgument('contest'))) {
```

Comment:
I can't attach the comment in unchanged lines, but please add a comment what the return value means.

Also verbosity is spelled wrong above.